calendar.html

```diff
@@ -15,10 +15,21 @@
 ---
 
 <div class="calendar-controls">
-      <button class="calendar-button hideable" id="css-calendar-switch">Switch View</button>
-      <a class="calendar-button" href="{{ page.calendar_url }}">Add to own Calendar</a>
+  <button class="calendar-button hideable" id="css-calendar-switch">Switch View</button>
+  <a class="calendar-button" href="{{ page.calendar_url }}">Add to own Calendar</a>
 </div>
 
 <div class="calendar-container">
+  <div class="calendar-event" id="calendar-event">
+      
+      <h2 class="event-tag event-tag-title">Event:</h2> <span class="event-text" id="event-text-title"></span><br>
```

Comment:
No need to have `.event-tag .event-tag-title`.
Instead I think we should prefer something like `.calendar-event .event-title`. We should try and keep the number of classes used to a minimum so that it's easier to work through and understand.
However, this isn't critical, it's just to keep the code easier to read and understand in the future.

Reply:
Removed redundant `event-tag-xxxx` tags. Removed line space on line 24